### PR TITLE
sonar smells

### PR DESCRIPTION
**Pull request by jeswanke**
Signed-off-by: John Swanke <jswanke@redhat.com>

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <html lang="en">
     <head>
         <meta charset="utf-8" />
-        <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+        <!-- <link rel="icon" type="image/svg+xml" href="favicon.svg" /> -->
```
Comment (KevinFCormier) [inline frontend/public/index.html:7]: Why this change, @jeswanke?

Comment (jeswanke) [reply frontend/public/index.html:7]: kept coming up in console that it didn't exist but looks like it exists in release so put it back
